### ISSUE TITLE
If there is a comment after that last item in a list, an error is incorrectly reported.

### ISSUE THREAD

**Issue by tomprince** [flake8-commas/flake8-commas#18]
For example,

``` python
stuff = [
   'a',
   'b',
   # more stuff will go here
]
```

is reported as an error.

**Comment by graingert** [comment]
@tomprince can you check if this is fixed in 0.2.0?
